script.js

```diff
@@ -0,0 +1,43 @@
+
+
+API_KEY = "yTk65md4VUpLFHxJvS6bqFOcT31mDqFg"
+//api key from GHIPY 
+const formEl = document.querySelector('form');
+document.addEventListener("DOMContentLoaded", giphysearch);
+
+//eventListener added 
+function giphysearch() {
+  document.querySelector(".button").addEventListener("click", event => {

```

Comment:
really good way to show of showing the event listener in click functioning for the user ⭐